frontend/src/ol/staticLayers/layerTypes/regions.js

```diff
@@ -16,7 +16,7 @@ const regionsLayer = (layerData, handleIsLoading) => {
     style: regionStyleFactory(layerData.attribute, layerData.style),
     legend: layerData.legend,
     zIndex: 0,
-    timeseries: layerData.timeseries,
+    layerOptions: layerData.layerOptions,
```

Comment:
We have 5 layerTypes, should I add this also to combined and geometry layers that didn't have timeseries? If maxResolution should work, it needs to be added here like:
maxResolution: layerData.layerOptions.maxResolution,
should I do it? What will I do with layerOptions: layerData.layerOptions then? I will remove maxResolution from it here?

And in group layer(groups.js) I need to send this maxResolution value to every layer as parameter. Because now I have one common maxResolution for every layer in group layer, but that makes sense right?

Reply:
> We have 5 layerTypes, should I add this also to combined and geometry layers that didn't have timeseries? 
yes, it should be the same for all layers
>If maxResolution should work, it needs to be added here like: maxResolution: layerData.layerOptions.maxResolution, should I do it?
yes 
>What will I do with layerOptions: layerData.layerOptions then? I will remove maxResolution from it here?
I believe it can stay or is there reason to not have it there?

> And in group layer(groups.js) I need to send this maxResolution value to every layer as parameter. Because now I have one common maxResolution for every layer in group layer, but that makes sense right?
not sure if it should be at the top, I can imagine case when we want display just one of multiple layers for layer group at specified resolution. Can you move it in each sublayer?

Reply:
I fixed these according to teams conversation.